### DISCUSSION TITLE
How to apply formatting?

### DISCUSSION THREAD

**Discussion by SuperCuber** [Byron/google-apis-rs#285] [Q&A]
Say I want to set a cell as a tickbox. Is there a way to do this currently? From what I see, even if I update an existing tickbox's cell it turns into a string.

**Comment by Byron** [comment]
As this repository is for the code generator, it's impossible to provide answers to most (if not any) of the APIs it generates code for.

The only suggestion I have is to try narrow down the problem on your own and find out if its incorrect API usage or an issue with the way the code is generated. In the latter case an issue could be opened to see if a fix is possible.
